cmake/features.cmake

```diff
@@ -116,6 +116,8 @@ ie_option (ENABLE_GAPI_PREPROCESSING "Enables G-API preprocessing" ON)
 ie_option (ENABLE_MULTI "Enables MULTI Device Plugin" ON)
 ie_option (ENABLE_AUTO "Enables AUTO Device Plugin" ON)
 
+ie_option (ENABLE_BATCH "Enables Auto-Batching Plugin" ON)
```

Comment:
batch is not always associated with plugin, maybe `ENABLE_AUTO_BATCH`?

Reply:
yep